maltoolbox/attackgraph/attackgraph.py

```diff
@@ -150,11 +150,28 @@ def _process_step_expression(
                     step_expression['stepExpression'])
                 new_target_assets.extend(assets)
 
-            selected_new_target_assets = [asset for asset in \
-                new_target_assets if specification.extends_asset(
-                    lang_graph._lang_spec,
-                    asset.type,
-                    step_expression['subType'])]
+            selected_new_target_assets = []
+            for asset in new_target_assets:
```

Comment:
Just a thought, does a language have assets or should we call them asset_types instead?
Assets are in my understanding part of models, and they are created from the asset types defined in the language.  This is nothing related to this PR in particular, just about the terminology used in the LanguageGraph.

Reply:
Yes, I had the exact same feeling when working on the language graph changes that the fact that both are called assets is very confusing. I think this will get resolved if we end up using the language graph to create the attack graph. So, it's a good observation, but we will see how it gets resolved if/when we get to that pull request.